### PR TITLE
Update about-admin-roles.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated service admin to Service Support Admin

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3090

File: microsoft-365/admin/add-users/about-admin-roles.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ You'll probably only need to assign the following roles in your organization. By
...
 |Office Apps admin    |   Assign the Office Apps admin role to users who need to do the following: <br> - Use the Office cloud policy service to create and manage cloud-based policies for Office <br> - Create and manage service requests <br> - Manage the What's New content that users see in their Office apps   <br> - Monitor service health  |
-|Service admin    |   Assign the Service admin role as an additional role to admins or users whose role doesn't include the following, but still need to do the following: <br> - Open and manage service requests <br> - View and share message center posts   |
+|Service Support Admin   |   Assign the Service Support Admin role as an additional role to admins or users whose role doesn't include the following, but still need to do the following: <br> - Open and manage service requests <br> - View and share message center posts   |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/about-admin-roles.md]:
```suggestion
|Service Support admin   |   Assign the Service Support admin role as an additional role to admins or users whose role doesn't include the following, but still need to do the following: <br> - Open and manage service requests <br> - View and share message center posts   |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3003

**Pull request merged**